README.md

```diff
@@ -85,6 +85,16 @@ generally contain namespaced classes. These are exempted from `automatic_namespa
 * javascript
 * views
 
+Additional directories can be excluded by adding them to the `automatic_pack_namespace_exclusions` key in your 
```

Comment:
What do you think about adding some of the rationale for this feature here? Help the reader see the value of the capability.

Reply:
Let me know if what I put seems sufficient.